tabs/src/components/sample/QueryRunner.jsx

```diff
@@ -114,6 +122,7 @@ export function QueryRunner() {
 
     useEffect(() => {
         setQuery(GRAPH_URL + graphVersion + query.substring(GRAPH_URL.length + graphVersion.length, query.length));
+    // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [graphVersion, query]);
```

Comment:
![image](https://user-images.githubusercontent.com/55033656/128365580-58c58fdb-fcd8-43ed-bb4b-b94f01f3a686.png)

to remove the linting error

Reply:
or u can ignore the error, u can make the decision